ds_judgements_public_ui/static/css/includes/ncn_not_found.css

```diff
@@ -0,0 +1,12 @@
+.tna-message{
```

Comment:
I think the filename of this should be something more generic since it's not actually NCN not found specific, perhaps?

Reply:
In hindsight I think `tna-message` this might literally be the name of this component -- the message component from the tna design system.
https://nationalarchives.github.io/design-system/components/message/

So `copied-from-tna-design-system.css` ?